src/screens/TopicPageScreen/index.js

```diff
@@ -396,10 +401,11 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <View>
       <NavHeader
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `NavHeader` component now receives two additional props: `isLoading` and `opacityImage`. These changes seem to be part of the improvements mentioned in the PR summary. Ensure that the `NavHeader` component properly handles these new props.
```diff
- if (isInitialLoading) return null;
+ <NavHeader
+   isLoading={isInitialLoading}
+   opacityImage={opacityImage}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->